docker/synckeys/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.15 AS builder
+
+ENV GOOS linux
+ENV GOARCH amd64
+ENV CGO_ENABLED 0
+
+ADD synckeys.go /go/src/synckeys/
+RUN go install ./...
+
+FROM scratch
+
+COPY --from=builder /go/bin/synckeys /usr/local/bin/synckeys
```

Comment:
if this container exits after it's done then maybe we could drop the scratch image and just run from golang image?

Reply:
The point of the scratch image is that the resulting container is about 6MB instead of 600MB. The golang image is just a builder that creates a single binary that sits alone in the resulting container.